src/components/PostList/PostList.container.tsx

```diff
@@ -82,17 +85,17 @@ export const PostsListContainer: React.FC<PostsListContainerProps> = props => {
       </Grid>
     );
 
-  if (empty)
-    return (
-      <EmptyResult
-        emptyContent={
-          (query.type && query.type === TimelineType.EXPERIENCE) ||
-          (!query.type && user)
-            ? EmptyContentEnum.DISCOVER
-            : EmptyContentEnum.POST
-        }
-      />
-    );
+  // if (empty)
```

Comment:
just delete this, do not comment line of code, we can use git history for backup